scipy/linalg/_basic.py

```diff
@@ -970,21 +982,13 @@ def det(a, overwrite_a=False, check_finite=True):
     """
     Compute the determinant of a matrix
 
-    The determinant of a square matrix is a value derived arithmetically
-    from the coefficients of the matrix.
-
-    The determinant for a 3x3 matrix, for example, is computed as follows::
-
-        a    b    c
-        d    e    f = A
-        g    h    i
-
-        det(A) = a*e*i + b*f*g + c*d*h - c*e*g - b*d*i - a*f*h
+    The determinant is the scalar that is a function of the associated square
+    matrix coefficients. The scalar is zero for singular matrices.
```

Comment:
My English failed here probably.

Reply:
How about something like:
```suggestion
    The determinant is the product of the eigenvalues of a square matrix.
    For other interpretations of the determinant, see [1]_.
```

Reply:
For determinant I think a reference is an overkill. Since the reference will be wikipedia anyways and that would be basic googling. Product of eigenvalues is not how we compute it so we shouldn't make that connection in the definition.